### ISSUE TITLE
Output polluted when resetting a user password via occ and it does not fit the password policy

### ISSUE THREAD

**Issue by davitol** [owncloud/password_policy#48]
### Steps

1. Login as admin
2. Enable rule password must be different than your previous 3 passwords
3. Create a user "user1" with password "pwd"
4. Login as "user1"
5. Run `./occ user:resetpassword user1` and set the same password "pwd"

![screen shot 2018-07-13 at 09 23 07](https://user-images.githubusercontent.com/10596560/42678159-62b6410c-867e-11e8-8538-6276442daab8.png)

![screen shot 2018-07-13 at 08 44 01](https://user-images.githubusercontent.com/10596560/42676869-075942ea-867a-11e8-86fd-8d51e6166c85.png)

**Comment by PVince81** [comment]
on the other, if we do it like on Linux, an admin is allowed to set passwords that don't match the rules but get only a warning

**Comment by PVince81** [comment]
does user management let the admin set passwords from history ?

**Comment by phil-davis** [comment]
No, it gives back the same messages - enforces password policies (length...) and history. Although the admin should be very unlikely in practice to "accidentally" set a new password for a user that happens to be one of the user's old passwords.
